### ISSUE TITLE
If I create a symbol link to act-new.sh in my bin folder and run that link it will fail

### ISSUE THREAD

**Issue by greenlaw110** [actframework/act-starter-kit#1]
The error message:

```
luog@luog-Satellite-P50-A:~/p/tmp$ act-new 
[3;J
---------------------------------
Act.Framework Project Starter Kit
---------------------------------

(C)2017 Thinking.Studio 
Written by J.Cincotta

use -? for help.

Act.Framework Interactive Project Setup...

Enter the directory for the new project: /home/tmp/p/x

Enter the name of this project (lower case name): x

Enter the organisation package name (Java style like com.google.utilities): com.x

Would you like to use the Rythm templating engine with this project? [y/n]: Y
Would you like to use Node.JS and Gulp with this project? [y/n]: nWorking.mkdir: cannot create directory ‘/home/tmp/p/x’: No such file or directory
/home/luog/bin_local/act-new: line 118: cd: /home/tmp/p/x: No such file or directory
.mkdir: cannot create directory ‘src’: File exists
/home/luog/bin_local/act-new: line 123: cd: /home/tmp/p/x/src/: No such file or directory
.mkdir: cannot create directory ‘assembly’: File exists
cp: cannot stat '/home/luog/bin_local/rsrc/assembly/*': No such file or directory
./home/luog/bin_local/act-new: line 133: cd: /home/tmp/p/x/src/: No such file or directory
mkdir: cannot create directory ‘main’: File exists
./home/luog/bin_local/act-new: line 138: cd: /home/tmp/p/x/src/main/: No such file or directory
mkdir: cannot create directory ‘bin’: File exists
cp: cannot stat '/home/luog/bin_local/rsrc/bin/run': No such file or directory
chmod: cannot access './run': No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/bin/start': No such file or directory
chmod: cannot access './start': No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/bin/run.bat': No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/bin/start.bat': No such file or directory
./home/luog/bin_local/act-new: line 150: cd: /home/tmp/p/x/src/main/: No such file or directory
mkdir: cannot create directory ‘resources’: File exists
./home/luog/bin_local/act-new: line 156: cd: /home/tmp/p/x/src/main/resources/: No such file or directory
mkdir: cannot create directory ‘asset’: File exists
mkdir: cannot create directory ‘css’: File exists
mkdir: cannot create directory ‘js’: File exists
mkdir: cannot create directory ‘img’: File exists
./home/luog/bin_local/act-new: line 165: cd: /home/tmp/p/x/src/main/resources/: No such file or directory
mkdir: cannot create directory ‘conf’: File exists
mkdir: cannot create directory ‘common’: File exists
./home/luog/bin_local/act-new: line 174: cd: /home/tmp/p/x/src/main/resources/: No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/resources/app.version': No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/resources/logback.xml': No such file or directory
./home/luog/bin_local/act-new: line 184: cd: /home/tmp/p/x/src/main/: No such file or directory
mkdir: cannot create directory ‘java’: File exists
mkdir: cannot create directory ‘com’: File exists
sed: can't read /home/luog/bin_local/rsrc/java/App.java: No such file or directory
./home/luog/bin_local/act-new: line 236: cd: /home/tmp/p/x/: No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/root/run_dev': No such file or directory
chmod: cannot access './run_dev': No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/root/run_prod': No such file or directory
chmod: cannot access './run_prod': No such file or directory
cp: cannot stat '/home/luog/bin_local/rsrc/root/setup_project': No such file or directory
chmod: cannot access './setup_project': No such file or directory
.  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   791  100   791    0     0    450      0  0:00:01  0:00:01 --:--:--   450
sed: can't read /home/luog/bin_local/rsrc/root/pom.xml: No such file or directory
OK
```

**Comment by CINJ** [comment]
Are you able to let me know how you used the ln command? As in, did the soft-link use ~ in it's path?

**Comment by greenlaw110** [comment]
using `~` in project path is another issue. I've updated the description with another run without `~`.

So here is what I've done:

1. `ln -s ~/act-starter-kit/act-new.sh ~/bin_local/act-new`
2. `cd ~/tmp/p`
3. `act-new` - note the path `~/bin_local` in in my `$PATH` environment variable.

Then I entered the interactive mode of starter kit and the following scripts are all put into the issue description
